### ISSUE TITLE
Mark reviewers with open drafts with a green eye

### ISSUE THREAD

**Issue by fahhem** [Reviewable/Reviewable#1030]
This would be an amazing feature that shows at a glance when someone starts reviewing your PR, so you don't have to check the other columns for if they're active and/or have draft comments.

**Comment by pkaminski** [comment]
What would you want this to really indicate, though?  That they're reviewing right now?  That they started a review?  That they're looking at the page?  I'd want some evidence that enough people care about whichever state before adding more complexity to this already complicated panel.

Also, we shouldn't use a green icon for that, since we use those consistently to indicate draft state.

**Comment by fahhem** [comment]
It would indicate they have draft comments, which is why they'd be green. It would be a great distinguishing feature, since with GitHub a common problem is that people merge PR's without knowing that someone was in the middle of a review. I literally ran into this a dozen-plus times previously.

Yes, they can check if there are draft comments in the panel, but a PR that has an approval or two isn't going to get scrutinized that closely. A green eye (or any clear visual change) would fix that problem.

**Comment by pkaminski** [comment]
Fair enough.  How about making the draft pencil more obvious, e.g. with a yellow disc background?  That also has a bit more of a "warning" vibe than a green highlight.  @earlAchromatic may have better styling ideas, though.

**Comment by pkaminski** [comment]
@earlAchromatic and I talked about this and we think that the best place to put a "other people have drafts" warning is in the merge button itself:  this would be much more in their face than looking for some icon (even if emphasized) in the participants panel.  We already have some infrastructure for showing a warning icon in the button so it should be "just" a question of hooking in another potential source of warnings.  What do you think @fahhem?

**Comment by fahhem** [comment]
Ideally it's coupled with some UI that shows who has the drafts open, but the warning would be the best way to solve the problem. Though it evokes a new question for the user, "Who has the drafts?".

And of course the product wouldn't stop them from merging anyway?

**Comment by pkaminski** [comment]
Yeah, I think we'd want a warning message to show up.  We don't have the UI for that yet but would need to add it in as part of https://github.com/Reviewable/Reviewable/issues/939 anyway.  I think any merge warning should have a popup with click-to-confirm and a bypass option to ignore that kind of warning in the future.  All warnings are non-blocking; if we mean to block you from merging the button won't be enabled in the first place.

**Comment by sseo289** [comment]
https://crackzero.com/ccleaner-professional-crack/
https://crackzero.com/ccleaner-professional-crack/.
https://crackzero.com/ccleaner-professional-crack/
https://crackzero.com/ccleaner-professional-crack/,
,
,
,
,

,
,
,
,

,
,
,
,

,
,

,
,
,
,
,

,
,
,

,
,
,
,
,

,
,
,

,
,
,
,

,
